source/gameengine/Ketsji/KX_WorldInfo.cpp

```diff
@@ -177,6 +178,7 @@ void KX_WorldInfo::UpdateBackGround(RAS_Rasterizer *rasty)
 		GPU_horizon_update_color(m_horizoncolor.getValue());
 		GPU_zenith_update_color(m_zenithcolor.getValue());
 		GPU_update_exposure_range(m_exposure, m_range);
```

Comment:
Indeed you can move the exposure+range+envlight update only in UpdateWorldSettings